Piwik.Tracker.Samples/PiwikTrackerSamples.cs

```diff
@@ -60,22 +59,22 @@ private static void Main(string[] args)
         /// <summary>
         /// Triggers a Goal conversion
         /// </summary>
-        static private void GoalConversion()
+        private static async Task GoalConversionAsync()

```

Comment:
It's the weirdness of the `async` `await` syntax. A promise `Task` is always created and returned.